Pod/Classes/DottedLineComponent/ARDottedLineManager.m

```diff
@@ -0,0 +1,55 @@
+#import "ARDottedLineManager.h"
+
+
+
+@interface ARDottedLine : UIView
+@property (nonatomic, strong, readwrite) UIColor *color;
+@end
+
+
+@implementation ARDottedLine
+
+- (void)drawRect:(CGRect)rect
+{
+    [super drawRect:rect];
+    
+    [[UIColor whiteColor] setFill];
```

Comment:
I wonder if this should be `clearColor` instead and that’ll show the white background underneath. If it doesn’t then this is fine too 👍